proto/ibc/applications/transfer/v1/event.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+package ibc.applications.transfer.v1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types";
+
+import "gogoproto/gogo.proto";
+import "ibc/core/channel/v1/channel.proto";
+
+// EventOnRecvPacket is a typed event emitted on receiving packet
+message EventOnRecvPacket {
```

Comment:
Have a feeling this might want to be a `channel` event? cc @AdityaSripal @cwgoes

Reply:
yea this should be under `proto/ibc/core/channel`